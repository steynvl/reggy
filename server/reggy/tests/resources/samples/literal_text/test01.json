{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Literal text",
        "markedStrings": [
          "adsa"
        ],
        "markerInfo": {
          "caseInsensitive": false,
          "matchAllExceptSpecified": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
