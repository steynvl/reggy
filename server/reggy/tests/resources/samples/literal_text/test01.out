Java
Pattern regex = Pattern.compile("adsa");
adsa
Perl
my $regex = /adsa/;
adsa
POSIX
adsa
adsa
Python
regex = re.compile(r'adsa')
adsa
JavaScript
const regex = /adsa/;
adsa
PHP
$regex = '/adsa/';
adsa
Golang
regex, _ := regexp.Compile("adsa")
adsa
Rust
let re = Regex::new(r"adsa").unwrap();
adsa
C#
var re = new Regex(@"adsa");
adsa
Scala
val re = "adsa".r
adsa
Kotlin
val regex = Regex("adsa")
adsa